Source/Common/CPUInfo.cpp

```diff
@@ -11,14 +12,16 @@
 #include <linux/limits.h>
 #endif
 
-namespace FEXCore::CPUInfo {
+namespace FEX::CPUInfo {
 #ifndef _WIN32
 uint32_t CalculateNumberOfCPUs() {
-  char Tmp[PATH_MAX];
+  constexpr auto parse_string = FMT_COMPILE("/sys/devices/system/cpu/cpu{}");
+  constexpr auto max_parse_size = ::fmt::formatted_size(parse_string, std::numeric_limits<uint32_t>::max());
```

Comment:
Since it's a non-generic context, this could use UINT32_MAX just to be a tad less verbose

```suggestion
  constexpr auto max_parse_size = ::fmt::formatted_size(parse_string, UINT32_MAX);
```